frontend/src/scenes/UpgradeModal.tsx

```diff
@@ -39,7 +50,14 @@ export function UpgradeModal(): JSX.Element {
                 <b>{featureName && capitalizeFirstLetter(featureName)}</b> is an advanced PostHog feature.
             </p>
             {featureCaption && <p>{featureCaption}</p>}
-            <p className="mb-0">Upgrade and get access to this, as well as to other powerful enhancements.</p>
+            <p className="mb-0">
+                {featureFlags[FEATURE_FLAGS.BILLING_UPGRADE_LANGUAGE] === 'subscribe'
+                    ? 'Subscribe'
+                    : featureFlags[FEATURE_FLAGS.BILLING_UPGRADE_LANGUAGE] === 'credit_card' && !billing?.customer_id
```

Comment:
I feel like this check for whether they have an associated stripe id and therefore have added a credit card before is going to make feature flag analysis complicated. I think we should change this (and all other instances, of course) to say "add credit card" if they don't have a customer_id and "link existing credit card" if they do.

Reply:
Yeah I think it should be:

- If they do not have active sub:
  - "Subscribe" / "Upgrade" / "Add credit card"
- If they do have active sub:
  - "Subscribe" / "Upgrade" / "Add (to plan?)"

Because of the different cohorts here we should make sure we can read results reliably.

Reply:
Decided to say "add paid plan" on the credit card flag w/ subscription condition.